h/static/scripts/widget-controller.js

```diff
@@ -193,6 +203,12 @@ module.exports = function WidgetController(
     return annotation.$$tag in $scope.focusedAnnotations;
   };
 
+  $scope.selectedAnnotationUnavailable = function () {
+    return searchClients.length === 0 &&
```

Comment:
Related to a comment I made on #3108, relying on `searchClients.length` to determine whether requests are in-flight to fetch annotations is pretty opaque. At the very least add a comment, or better add a helper function elsewhere in this file to hide the details of `searchClients` from other parts of the code.